kolibri/plugins/coach/assets/src/views/lessons/LessonSummaryPage/sidePanels/SelectedResource.vue

```diff
@@ -0,0 +1,158 @@
+
+    <div
+      v-for="(lesson, index) in resources"
+      :key="index"
+    >
+      <KGrid :style="lessonOrderListButtonBorder">
+        <KGridItem :layout12="{ span: 6 }">
+          <div style="display: flex">
+            <div>
+              <DragSortWidget
+                moveUpText="up"
+                moveDownText="down"
+                :noDrag="true"
+                :isFirst="index === 0"
+                :isLast="index === resources.length - 1"
+                @moveUp="() => {}"
+                @moveDown="() => {}"
+              />
+            </div>
+            <div style="padding: 0 10px">
+              <span style="flex: 1">
+                <LearningActivityIcon
+                  :kind="lesson.learning_activities[0]"
+                  class="icon-style"
+                />
+              </span>
+            </div>
+            <div>
+              <span class="arrange-item-block">
+                <span>
+                  <KRouterLink
+                    v-if="lesson.link"
+                    :text="lesson.title"
+                    :to="lesson.link"
+                    style="font-size: 14px"
+                  />
+                </span>
+                <p style="font-size: 12px">4MB</p>
+              </span>
+            </div>
+          </div>
+        </KGridItem>
+
+        <KGridItem :layout12="{ span: 6 }">
+          <span class="add-minus-button">
+            <KIconButton
+              icon="emptyTopic"
+              @click="() => {}"
+            />
+
+            <KIconButton
+              icon="minus"
+              @click="removeResource(lesson.id)"
+            />
+          </span>
+        </KGridItem>
+      </KGrid>
+    </div>
+  </div>
+
+</template>
+
+
+<script>
+
+  import DragSortWidget from 'kolibri-common/components/sortable/DragSortWidget';
+  import commonCoreStrings from 'kolibri/uiText/commonCoreStrings';
+  import LearningActivityIcon from 'kolibri-common/components/ResourceDisplayAndSearch/LearningActivityIcon.vue';
+  import bytesForHumans from 'kolibri/uiText/bytesForHumans';
+
+  export default {
+    name: 'SelectedResource',
+    components: {
+      DragSortWidget,
+      LearningActivityIcon,
+    },
+    mixins: [commonCoreStrings],
+    props: {
+      resourceList: {
+        type: Array,
+        required: true,
+      },
+      currentLesson: {
+        type: Object,
+        required: true,
+      },
+      loading: {
+        type: Boolean,
+        required: true,
+      },
+    },
+    data() {
+      return {
+        resources: [],
+      };
+    },
+    computed: {
+      lessonOrderListButtonBorder() {
+        return {
+          borderBottom: `1px solid ${this.$themePalette.grey.v_200}`,
+          height: `4em`,
+          marginTop: `0.5em`,
+        };
+      },
+    },
+    mounted() {
+      setTimeout(() => {
+        this.resources = this.resourceList;
+      }, 1000);
```

Comment:
This is causing you to have an "extra" loading state, and it's mismatched with your loading spinner condition. I'm not sure if this is also here for testing, or if there is something else happening with loading/state management that you're trying to work around, but if that's the case, let's chat more about the situation and see if we can find another solution